doc/api/errors.md

```diff
@@ -826,16 +826,6 @@ when an error occurs (and is caught) during the creation of the
 context, for example, when the allocation fails or the maximum call stack
 size is reached when the context is created.
 
-<a id="ERR_CPU_PROFILE_ALREADY_STARTED"></a>
```

Comment:
We probably shouldn't delete the error right away. A key reason for these being documented here is to make it easy for users to find info if they encounter the error. Even if the error is not used any more in newer versions they should be able to find it still.